### ISSUE TITLE
[gh-pages] Add a timestamp

### ISSUE THREAD

**Issue by brunocodutra** [ldionne/metabench#62]
Just a reminder so we don't forget to add a timestamp somewhere (on the footer?) informing at what time results were last updated.

**Comment by ldionne** [comment]
The following snippet will add the date the website was last generated:

```
Generated on {{ site.time | date: '%a, %-d %b %Y' }}
```

The formatting of the date is explained [here](https://docs.shopify.com/themes/liquid/filters/additional-filters#date). I suck so much at HTML that I'd rather let you find the best place to put this. I think just above or just below the Copyright statement would be OK, but if you have better ideas then I'm OK with this too.

**Comment by brunocodutra** [comment]
> I'd rather let you find the best place to put this

How about we place it right below the top bar on the right, but spell it like the following instead?

```
Last updated on Thu, 24 Mar, 2016
```

I'll be on a trip for holidays this weekend, so I'll won't be able to work on it until monday next week, but you can assign this to me.

**Comment by ldionne** [comment]
> How about we place it right below the top bar on the right, but spell it like the following instead?

Works for me.

> I'll be on a trip for holidays this weekend, so I'll won't be able to work on it until monday next week, but you can assign this to me.

Alright!

**Comment by ldionne** [comment]
Fixed by #79 and #80.
